src/order/service.js

```diff
@@ -0,0 +1,27 @@
+const axios = require("axios");
+
+class OrderService {
+  constructor(apiUrl) {
+    this.apiUrl = apiUrl;
+  }
+
+  async getOrders(requestBody, signature, publisherToken) {
+    const response = await axios.post(
+      this.apiUrl + "/reporting/reports/orders",
+      requestBody,
+      {
+        headers: {
+          signature: signature,
```

Comment:
remove all signatures from requests to the gateways, publisher token should be enough

Reply:
I'm not sure we removed it from all our packages. I'll discuss it with @talkot1